planner/cascades/transformation_rules.go

```diff
@@ -992,3 +993,34 @@ func (r *MergeAggregationProjection) OnTransform(old *memo.ExprIter) (newExprs [
 	newAggExpr.SetChildren(old.Children[0].GetExpr().Children...)
 	return []*memo.GroupExpr{newAggExpr}, true, false, nil
 }
+
+// MergeAdjacentSelection merge adjacent selection
```

Comment:
```suggestion
// MergeAdjacentSelection merge adjacent selection.
```

nit: Comment should always be a complete sentence. So we'd better add a '.' at the end.

Reply:
got